west.yml

```diff
@@ -16,7 +16,7 @@ manifest:
           - nanopb
           - hal_st
     - name: orb-messages
-      revision: 0cf4636b99211cdbebba3948776cbb8c351d45af
+      revision: 61e2ad2a7567266c805bae6e3d10775eb2964db7
```

Comment:
the PR has been merged on the orb-messages side, let's track the commit in the main branch